### PR TITLE
chore: use file paths io url routes

### PR DESCRIPTION
**Pull request by stijnmoreels**
Use relative file paths instead of URL routes for more stable sub-page linking.

Related to https://github.com/arcus-azure/arcus/issues/191

### CODE REVIEW

**Review by fgheysels — APPROVED** [review]
Just some suggestions :)

File: docs/preview/features/logging.md [hunk 1]
```diff
@@ -348,7 +367,7 @@ public class Startup
 }
 ```
 
-This means that every endpoint will only track `201 Created` responses. Changing this in the options usually for when you want to straightline your entire application and only allow a certain set of status codes to be tracked.
+This means that every endpoint will only track `201 Created` responses. Changing this in the options usually for when you want to straighten your entire application and only allow a certain set of status codes to be tracked.
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/preview/features/logging.md:370]: Changed this in all versions, too.

File: docs/preview/features/logging.md [hunk 2]
```diff
@@ -348,7 +367,7 @@ public class Startup
...
 More grainer control can be achieved via placing an attribute on either the controller's class definition or the endpoint method:
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/preview/features/logging.md:371]: Changed in all versions, too.

File: docs/preview/features/openapi/security-definitions.md [hunk 3]
```diff
@@ -5,12 +5,15 @@ layout: default
 
 # Adding OAuth security definition to API operations
 
-When an API is secured via OAuth, [Shared Access Key authentication](../../features/security/auth/shared-access-key), [Certificate authentication](../../features/security/auth/certificate), it is helpful if the Open API documentation makes this clear via a security scheme and the API operations that require authorization automatically inform the consumer that it is possible that a 401 Unauthorized or 403 Forbidden response is returned.
+When an API is secured via OAuth, [Shared Access Key authentication](../security/auth/shared-access-key.md), [Certificate authentication](../security/auth/certificate.md), it is helpful if the Open API documentation makes this clear via a security scheme and the API operations that require authorization automatically inform the consumer that it is possible that a 401 Unauthorized or 403 Forbidden response is returned.
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/preview/features/openapi/security-definitions.md:8]: Yeah, it's both as we have authentication (shared access & certificate) and authorization (jwt). I'll add 'authentication' here, on all versions.

File: docs/preview/features/security/auth/shared-access-key.md [hunk 4]
```diff
@@ -11,18 +11,25 @@ This authentication process consists of two parts:
 1. Find the configured parameter that holds the shared access key, this can be a request header, a query parameter or both.
 2. Shared access key matches the value with the secret stored, determined via configured secret provider
 
-The package allows two ways to configure this type of authentication mechanmism in an <span>ASP.NET</span> application:
-- [Global Shared access key authentication](#globally-enforce-shared-access-key-authentication)
-- [Shared access key authentication per controller or operation](#enforce-shared-access-key-authentication-per-controller-or-operation)
-- [Behavior in validating shared access key parameter](#behavior-in-validating-shared-access-key-parameter)
-- [Bypassing authentication](#bypassing-authentication)
+The package allows two ways to configure this type of authentication mechanism in an <span>ASP.NET</span> application:
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/preview/features/security/auth/shared-access-key.md:14]: Hmm, I'll remove this line completely, as with the table of contents, this is more clearly shown.

File: docs/versioned_docs/version-v0.2.0/features/logging.md [hunk 5]
```diff
@@ -64,6 +64,6 @@ public class Program
 To be able to use the `AddApplicationInsights` extension method, the Microsoft.Extensions.Logging.ApplicationInsights package must be installed.
 
 
-If the parameter-less `AddApplicationInsights` method is used, the configurationsetting `ApplicationInsights:InstrumentationKey` must be specified and the value of the instrumentation-key of the Application Insights resource must be set.
+If the parameter-less `AddApplicationInsights` method is used, the configuration setting `ApplicationInsights:InstrumentationKey` must be specified and the value of the instrumentation-key of the Application Insights resource must be set.
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/versioned_docs/version-v0.2.0/features/logging.md:67]: Ah, hmm, this is probably something that has grown from appsettings and token replacement. Something we maybe can discuss separately.

File: docs/versioned_docs/version-v0.1.0/features/security/auth/shared-access-key.md [hunk 6]
```diff
@@ -11,9 +11,15 @@ This authentication process consists of two parts:
 1. Looks for the configured HTTP request header that contains the shared access key
 2. Shared access key matches the value with the secret stored, determined via configured secret provider
 
-The package allows two ways to configure this type of authentication mechanmism in an <span>ASP.NET</span> application:
-- [Global Shared access key authentication](#globally-enforce-shared-access-key-authentication)
-- [Shared access key authentication per controller or operation](#enforce-shared-access-key-authentication-per-controller-or-operation)
+The package allows two ways to configure this type of authentication mechanism in an <span>ASP.NET</span> application:
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/versioned_docs/version-v0.1.0/features/security/auth/shared-access-key.md]: Hmm, I'll remove this line completely, as with the table of contents, this is more clearly shown. Also removed this on the certificate authentication on all versions.

File: docs/versioned_docs/version-v0.2.0/features/logging.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (fgheysels) [reply docs/versioned_docs/version-v0.2.0/features/logging.md:67]: True, I've opened a separate issue.

**Pull request merged**